src/main/java/Command.java

```diff
@@ -0,0 +1,17 @@
+public class Command {
+    private CommandType commandType;
+
+    public CommandType getCommandType() {
+        return commandType;
+    }
+
+    public String[] getAdditionalParameters() {
+        return additionalParameters;
+    }
+
+    private String[] additionalParameters;
```

Comment:
It may be slightly clearer if all the variables are declared at the start of the document, since other people reading from top to bottom may be a little confused about what additionalParameters is.